bin/init_script.sh

```diff
@@ -10,6 +10,7 @@ exec > >(tee /var/log/user-data.log | logger -t user-data -s 2>/dev/console) 2>&
 APPLICATION=galaxy
 GALAXY_API_KEY=changeme
 PULSAR_API_KEY=changeme
+ANSIBLE_USER=azureuser
```

Comment:
```suggestion
ANSIBLE_USER=ubuntu
```

The default user can be set when launching a VM on Azure:
![image](https://github.com/user-attachments/assets/a84ee4c5-0cfe-42cb-86f1-6c2d117d93b5)

So we can leave this variable in case other providers do not have such an option but seems to be it should default to `ubuntu`.